run

```diff
@@ -89,6 +89,13 @@ EOF
   if [ "${RSYSLOG_LOG_TO_FILE}" == 'yes' ] ; then
     echo 'mail.* -/var/log/mail.log' >> /etc/rsyslog.conf
   fi
+
+  if [ ! ${RSYSLOG_REMOTE_PORT} ] ; then
+    RSYSLOG_REMOTE_PORT=514
+  fi
+  if [ ${RSYSLOG_REMOTE_IP} ] ; then
+    echo "*.* action(type=\"omfwd\" target=\"${RSYSLOG_REMOTE_IP}\" port=\"${RSYSLOG_REMOTE_PORT}\" template=\"RSYSLOG_SyslogProtocol23Format\")" >> /etc/rsyslog.conf
```

Comment:
I don't know much about syslog protocols, do ppl use other protocols like `RSYSLOG_ForwardFormat`?

Reply:
I have to check it and test first.  RSYSLOG_SyslogProtocol23Format  gave me nice output in Graylog, it has split message to separate columns out of the box, no need to parse it.
![image](https://user-images.githubusercontent.com/98548898/184323732-9660a218-a871-494a-ad1a-11b8918d9460.png)

Reply:
Ok, think i'm fine either hardcode or have an option with default RSYSLOG_SyslogProtocol23Format